engine/apps/alerts/grafana_alerting_sync_manager/grafana_alerting_sync.py

```diff
@@ -348,11 +349,7 @@ def _get_receiver_config_for_datasource(self, is_grafana_datasource, webhook_url
             }
         return receiver
 
-    def _create_contact_point_from_payload(
-        self,
-        payload,
-        datasource,
-    ) -> "apps.alerts.models.GrafanaAlertingContactPoint":
```

Comment:
`apps.alerts.models.GrafanaAlertingContactPoint` wasn't resolved properly for some reason. Explicitly importing `GrafanaAlertingContactPoint` above with:
```python
if TYPE_CHECKING:
    from apps.alerts.models import GrafanaAlertingContactPoint
```